pinot-core/src/main/java/org/apache/pinot/core/transport/ServerChannels.java

```diff
@@ -201,7 +201,7 @@ void sendRequestWithoutLocking(String rawTableName, AsyncQueryResponse asyncQuer
         ServerRoutingInstance serverRoutingInstance, byte[] requestBytes) {
       long startTimeMs = System.currentTimeMillis();
       _channel.writeAndFlush(Unpooled.wrappedBuffer(requestBytes)).addListener(f -> {
-        long requestSentLatencyMs = System.currentTimeMillis() - startTimeMs;
+        int requestSentLatencyMs = (int) (System.currentTimeMillis() - startTimeMs);
```

Comment:
Minor suggestion: in Line 205, the integer would still need to be converted to of long type. How about adding the `(int)` only in Line 207?

Reply:
I feel it is more readable this way (use `int` to represent delta). Don't think there will be performance difference after the code is compiled

Reply:
I thought one of the purposes of this PR is to improve performance of the query part (e.g. by allocating less heap mem on initiating a int), so that's why I suggest this as well, as it helps reduce one round of data type conversion. But I'm fine with the current code change in this PR. :)